### ISSUE TITLE
Array with children with default values.

### ISSUE THREAD

**Issue by jma** [ngx-formly/ngx-formly#3246]
**Description**
Having an array with children with a default value is not populated at the form creation.

**Minimal Reproduction**

https://stackblitz.com/edit/angular-zt5dys?file=src%2Fassets%2Fjson-schema%2Farrays.json.
The value "name" should be in the input field. 

**Your Environment**

- Angular version: 7
- Formly version: 5.11.12

**Comment by aitboudad** [comment]
The issue is in default value of listOfObjects:  `"default": [null],`, so either pass name default or an `undefined`  value:

* Solution 1
```json
"listOfObjects": {
  "default": ["name"],
}
```

** Solution 2
```json
"listOfObjects": {
  "default": [undefined],
}
```

**Comment by jma** [comment]
Thanks. Unfortunately `undefined` is not a valid JOSON value. Moreover the idea behind this is to prep populate an array with the number of elements is equal to the `minItems` property. The solution 1 is thus complex to create as the item array element can be a large structure with several default values.

**Comment by aitboudad** [comment]
> Unfortunately undefined is not a valid JSON value.

use map to assign `undefined` to `defaultValue` and remove `default` from "listOfObjects":

```ts
this.formlyJsonschema.toFieldConfig(schema, {
  map: (field: FormlyFieldConfig, schema: JSONSchema7) => {
    if (schema.minItems > 0 && !field.defaultValue) {
      field.defaultValue = ...
    }

    return field;
  },
});
```

> Moreover the idea behind this is to pre-populate an array with the number of elements is equal to the `minItems` property.

I see, we'll introduce that behavior in `V6`.

**Comment by aitboudad** [comment]
This issue has been fixed and released as part of [v6.0.0-beta.0](https://github.com/ngx-formly/ngx-formly/releases/tag/v6.0.0-beta.0) release.

Please let us know, in case you are still encountering a similar issue/problem.
Thank you!
